openrr-nav-viewer/src/main.rs

```diff
@@ -1,7 +1,31 @@
+use clap::Parser;
 use openrr_nav_viewer::*;
 
+#[derive(Debug, Parser)]
+struct Args {
+    #[clap(
+        short = 'f',
+        long = "config-file",
+        default_value = concat!(
+            env!("CARGO_MANIFEST_DIR"),
+            "/../openrr-nav/config/dwa_parameter_config.yaml"
+        ),
```

Comment:
```suggestion
```

Depending on how it is installed, the main.rs binary may not have access to this path (e.g., when installing from crates.io with `cargo install`), so it is better to remove default path (and update "how to use" comment in `openrr-nav-viewer/examples/controller.rs` to pass `-- openrr-nav/config/dwa_parameter_config.yaml`).